lib/elastic_record/index/scroll_enumerator.rb

```diff
@@ -0,0 +1,59 @@
+module ElasticRecord
+  class Index
+    class ScrollEnumerator
```

Comment:
This too is unchanged, just pulled out into its own file.

Reply:
It would have been easier to review if the code that was just moved around without changes was in a separate PR.  Consider breaking this into 2 PR's.